libs/src/utils/network.js

```diff
@@ -62,10 +62,40 @@ async function timeRequestsAndSortByVersion (requests, timeout = null) {
     // If health check failed, send to back of timings
     if (!a.response) return 1
     if (!b.response) return -1
+
     // Sort by highest version
     if (semver.gt(a.response.data.data.version, b.response.data.data.version)) return -1
     if (semver.lt(a.response.data.data.version, b.response.data.data.version)) return 1
-    // If same version, do a tie breaker on the response time
+
```

Comment:
this function is designed to apply for both CN and DN - for now i think we only use it for CN in protocol, but sauron uses it for DN as well
we're breaking the service-type-agnosticism of this function now - would be annoying to refactor so at very elast for safety we should pass in a flag `checkCNLoad` defaulted to false or something

Reply:
could you send me the link for this?

Reply:
https://github.com/AudiusProject/sauron/blob/master/hobbits/checkSPVersion.js#L14

Reply:
i just checked every call to this function, just seems unsafe to assume all callers will use this only for CN

Reply:
agreed

Reply:
reverted